packages/vaadin-item/test/item-mixin.test.js

```diff
@@ -120,18 +97,18 @@ describe('vaadin-item-mixin', () => {
       });
 
       it('should have active attribute on space down', () => {
-        spaceDown(item);
+        spaceKeyDown(item);
         expect(item.hasAttribute('active')).to.be.true;
       });
 
       it('should not have active attribute after space up', () => {
-        spaceDown(item);
-        spaceUp(item);
+        spaceKeyDown(item);
```

Comment:
Suggestion: `spaceKeyDown()` + `spaceKeyUp()` => `space()`

Reply:
I think we want to be explicit here, so IMO keeping `spaceKeyUp` in this case makes sense, as it's what we test.